main_routers/system_router.py

```
@@ -0,0 +1,858 @@
+"""
+
+import os
+import sys
+import asyncio
+import logging
+from urllib.parse import unquote
+
+from fastapi import APIRouter, Request
+from fastapi.responses import JSONResponse, Response
+from openai import AsyncOpenAI
+from fastapi.responses import JSONResponse
```

Ruff (0.14.8)

Redefinition of unused `JSONResponse` from line 19

Remove definition: `JSONResponse`

(F811)